vue/components/ui/logging-container-platforme/logging-container-platforme.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <div class="logging-container">
+
+      <div
+        v-for="entry in entries"
+        v-bind:key= entry.mesage>
+        <log-entry
+            v-bind:entry = entry.entry>
+        </log-entry>
+      </div>
+
+  </div>
+</template>
+
+<style>
+.logging-container {
+    background-color: #0d1d24;
+    color: #ffffff;
+    font-family: "Lucida Console", Monaco, monospace;
+    font-size: 15px;
+    height: 190px;
+    overflow: auto;
+    padding: 5px;
+    text-align: left;
+    width: 100%;
+}
+
+</style>
+
+<script>
+import Vue from "vue";
+
+export const logcontainer = Vue.component("logging-container-platforme", {
+    data(){
```

Comment:
This should be a prop instead. Your example will soft-code this dummy data but if this is hard-coded here clients won't be able to override it.